assets/src/console/pages/products/ListPage.tsx

```diff
@@ -24,6 +25,34 @@ export interface props {
   gatewayAccount: GatewayAccountFragment;
 }
 
+const Ul = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 0;
+`;
+
+const Li = styled.li`
+  margin: 1rem 0 3rem;
+  ${P} {
+    font-weight: 700;
+    margin-bottom: 0;
+  }
+`;
+
+const RedLink = styled(Link)`
```

Comment:
does `RedLink` indicate that this action is dangerous and caution is required or something else? if so perhaps `DangerousLink` is better so that the colour red is not tied into the component name—colour can change, intention shouldn't.

Reply:
@jonathaningram - It indicates that text & link in red colour.  We are using this to display "Edit & Delete" links. To differentiate from usual "Link" which is blue in colour i have named it as RedLink.

Reply:
it's better to name components for their behaviour rather than their appearance. along the same times as having primary and secondary buttons versus having purple and grey buttons—the colour can change but the behaviour/function should not. so in this case `DangerousLink` is probably a better name. may not be the perfect name, but it's probably better than `RedLink`.